contrib/init/systemd/docker.service

```diff
@@ -5,7 +5,8 @@ After=network.target docker.socket
 Requires=docker.socket
 
 [Service]
-ExecStart=/usr/bin/docker -d -H fd://
+EnvironmentFile=/etc/default/docker
```

Comment:
Even that doesn't -- what about "/etc/conf.d/docker" ?  And what about the
fact that systemd upstream discourages this?

And what about the fact that "export http_proxy=..." won't work properly
as-is with this?

I think we're setting our users up for disappointment here, especially now
that we've got some documentation for how to adjust this properly.